traffic_ops/testing/api_contract/v4/test_cdns.py

```diff
@@ -0,0 +1,68 @@
+"""Api Contract Test Case for cdns endpoint"""
+import json
+import logging
+import pytest
+
+# Create and configure logger
+logger = logging.getLogger()
+
+
+@pytest.fixture(name="get_cdn_data")
+def get_cdn_prereq_data():
```

Comment:
missing return type on function

Reply:
Added return type for get_cdn_prereq_data().